apps/wallet/src/ui/index.tsx

```diff
@@ -74,11 +74,11 @@ function AppWrapper() {
 								},
 							}}
 						>
-							<RpcClientContext.Provider value={api.instance.fullNode}>
+							<SuiClientProvider networks={{ [api.apiEnv]: api.instance.fullNode }}>
```

Comment:
unlike explorer, in wallet the client instance was an external instance.  This also seems to work fine, but makes me wonder if maybe the context provider shouldn't control the instance of the client, and just expect you to manage it in your own.

The only thing we really gain by managing it in the provider today is a lot of complexity, and the query pre-fix.  I think if we just expect you to provide the network name or something in the provider things get a lot simpler... what do you think?